### ISSUE TITLE
Results using config_real structure IAM

### ISSUE THREAD

**Issue by josarajar** [rwth-i6/returnn#33]
Hello, thanks for RETURNN Framework, it is really useful.

I've trained the network structure in https://github.com/rwth-i6/returnn/blob/master/demos/mdlstm/IAM/config_real with the train and dev datasets generated with the code in _create_IAM_dataset.py_ and it gets LER of 5.4 % in the validation set (I see it in the log file) and LER of 8.8% when test set it is computed with config_fwd, and externally decoded and evaluated. 
However, the results showed in https://www.vision.rwth-aachen.de/media/papers/MDLSTM_final.pdf, indicate CER = 2.4 % and 3.5 % for dev and eval sets respectively.
Did you get these results with this code directly or did you use the word-based trigram and character language model indicated in section III in that paper (that I guess it is not implemented in the code)?

Thank you in advance.

**Comment by pvoigtlaender** [comment]
Hi, for the paper we used an decoder with lexicon, language model, search algorithm, and open vocabulary modeling. All these components are not part of RETURNN, but of RASR (https://www-i6.informatik.rwth-aachen.de/rwth-asr/)
I think, not all parts of the open vocabulary approach are contained in the publicly available version of the code.

**Comment by josarajar** [comment]
Thank you for answering. I'll try RASR.

**Comment by albertz** [comment]
Closing this as it is not really a bug in Returnn.

**Comment by prolaser** [comment]
Dear @josarajar 

I am a beginner in this field. I have trained my model until 72 epochs and the i ran the config_fwd which gave me another h5 file (mdlstm_real_valid.h5). could you please tell me how did you calculate the LER and CER based on these given files? I will really appreciate your help. are there any scripts which can help me to calculate the accuracy of my models or CER.
Thank you
